logsmanagement/README.md

```diff
@@ -0,0 +1,676 @@
+
+|  Configuration Option | Description  |
+|      :------------:  	| ------------ |
+| `log path` | Path to the systemd journal directory. If set to `auto`, the default path will be used to read local-only logs. |
+| `priority value chart` | Enable chart showing Syslog Priority values (PRIVAL) of collected logs. The Priority value ranges from 0 to 191 and represents both the Facility and Severity. It is calculated by first multiplying the Facility number by 8 and then adding the numerical value of the Severity. Please see the [rfc5424: Syslog Protocol](https://www.rfc-editor.org/rfc/rfc5424#section-6.2.1) document for more information.|
+| `severity chart` | Enable chart showing Syslog Severity values of collected logs. Severity values are in the range of 0 to 7 inclusive.|
+| `facility chart` | Enable chart showing Syslog Facility values of collected logs. Facility values show which subsystem generated the log and are in the range of 0 to 23 inclusive.|
+
+<a name="collector-configuration-docker-events"/>
+
+### Docker events
+
+</a>
+
+This collector will use the Docker API to collect Docker events logs. See also documentation of [Fluent Bit docker events input plugin](https://docs.fluentbit.io/manual/pipeline/inputs/docker-events).
+
+|  Configuration Option | Description  |
+|      :------------:  	| ------------ |
+| `log path` | Docker socket UNIX path. If set to `auto`, the default path will be used. |
+| `event type chart` | Enable chart showing the Docker object type of the collected logs. |
+| `event action chart` | Enable chart showing the Docker object action of the collected logs. |
+
+<a name="collector-configuration-tail"/>
+
+### Tail
+
+</a>
+
+This collector will collect any type of logs from a log file, similar to executing the `tail -f` command. See also documentation of [Fluent Bit tail plugin](https://docs.fluentbit.io/manual/pipeline/inputs/tail).
+
+|  Configuration Option | Description  |
+|      :------------:  	| ------------ |
+| `log path` | The path to the log file to be monitored. |
+| `use inotify` | Select between inotify and file stat watchers (providing `libfluent-bit.so` has been built with inotify support). It defaults to `yes`. Set to `no` if abnormally high CPU usage is observed or if the log source is expected to consistently produce tens of thousands of (unbuffered) logs per second. |
+
+<a name="collector-configuration-web-log"/>
+
+### Web log
+
+</a>
+
+This collector will collect [Apache](https://httpd.apache.org/) and [Nginx](https://nginx.org/) access logs.
+
+|  Configuration Option | Description  |
+|      :------------:  	| ------------ |
+| `log path` | The path to the web server's `access.log`. If set to `auto`, the collector will attempt to auto-discover it, provided the name of the configuration section is either `Apache access.log` or `Nginx access.log`. |
```

Comment:
I do not think we need to repeat the file `access.log` in both (ping @Ancairon )